.github/workflows/maintenance-renovate-theme-sync.yml

```
@@ -0,0 +1,94 @@
+        uses: ./.github/actions/setup-env
+        with:
+          node-version: '22'
+          skip-ruby: 'true'
+
+      - name: Run theme sync
+        run: bun run theme:sync
+
+      - name: Commit and push regen if needed
+        env:
+          APP_SLUG: ${{ steps.app-token.outputs.app-slug }}
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)